README.md

```diff
@@ -23,11 +23,52 @@ Your new file structure should look like:
     - Hermes
       - src
 
+## RabbitMQ Broker
+
+### Getting setup
+
+1) Installing rabbitMQ
```

Comment:
I think the instructions need to be more brain dead. We're gonna get alot of newbies who don't really know much of anything.